sdk/ai/azure-ai-projects/azure/ai/projects/telemetry/agents/_ai_agents_instrumentor.py

```diff
@@ -306,17 +304,17 @@ def _create_event_attributes(
 
         if cancelled_at:
             if isinstance(cancelled_at, datetime):
-                attrs[GEN_AI_RUNSTEP_CANCEL_TIMESTAMP] = cancelled_at.isoformat()
```

Comment:
we record status of step on line 289, no need to make it part of the attribute name